.cicd/create-docker-from-binary.sh

```diff
@@ -39,10 +39,12 @@ for REG in ${EOSIO_REGS[@]}; do
     eval $CLEAN_IMAGE_COMMIT 
     eval $CLEAN_IMAGE_BRANCH
     if [[ ! -z "$SANITIZED_TAG" ]]; then
-        DOCKER_TAG="docker tag eosio_image $REG:$SANITIZED_TAG"
+        DOCKER_TAG="docker tag eosio_image:$BUILD_TAG $REG:$SANITIZED_TAG"
+        DOCKER_PUSH_TAG="docker push $REG:$SANITIZED_TAG"
         DOCKER_REM="docker rmi $REG:$SANITIZED_TAG"
```

Comment:
Let's change this...
```bash
DOCKER_REM="docker rmi $REG:$SANITIZED_TAG"
```
...to include an "or no-op" so it never fails on the cleanup step again:
```bash
DOCKER_REM="docker rmi $REG:$SANITIZED_TAG || :"
```